### DISCUSSION TITLE
Enable instant battle by default

### DISCUSSION THREAD

**Discussion by oleg-derevenetz** [ihhub/fheroes2#6365] [Ideas]
I propose to enable the instant battle (without auto spell casting) in the upcoming 1.0 release by default (currently the default is manual battle, with auto spell casting enabled) to promote this new feature (sort of), because players (especially "old school" ones) usually have no idea that this feature exists, since it was completely absent from the original game.

**Comment by zenseii** [comment]
Hi, @oleg-derevenetz. I share the same thoughts, not just this feature but so many others like pressing Q in battle and moving all troops etc.

Anyways my main worry about enabling it by default is for the new players who would not even get a single incentive to learn how to battle.
This is why I've proposed earlier to have it always enabled for those battles when the opposing army flees or gets offended and attacks.

One solution to all of them would be to make a video showing all the new features fheroes2 has.

**Reply by oleg-derevenetz** [reply]
> Anyways my main worry about enabling it by default is for the new players who would not even get a single incentive to learn how to battle.

I believe that we need to give them a choice. If they don't want to do the battles manually, then so be it.

**Comment by oleg-derevenetz** [comment]
Hi @ihhub what do you think about this proposal?

**Reply by ihhub** [reply]
Hi @oleg-derevenetz and @zenseii ,

both points are equally correct. Ideally to solve this we need an in-game tutorial guiding a player thought all possible options. I am still thinking which side to choose :)

**Reply by ihhub** [reply]
@Branikolog , your call :)

**Comment by Branikolog** [comment]
Hi all.
I'm OK with both current and the proposed behaviour.
Originally, Heroes games I used to play, have this option disabled on default. I believe, any player should look through all options before playing the game. 
But on the same time the majority of players use instant battle, so enabling it on default is a good friendly approach.

**Reply by ihhub** [reply]
Hi @oleg-derevenetz , you have permission to enable it by default :)
